### ISSUE TITLE
feature request: case insencitive mode and float torrance

### ISSUE THREAD

**Issue by cubercsl** [xalanq/cf-tool#72]
Some problems may have specal judge, like "Yes" and "No" or float number compare.

But most time we use `cf test` testing these problems will fail.

So I think we can add the support for the testlib std [checkers](https://github.com/MikeMirzayanov/testlib/tree/master/checkers).

**Comment by infalmo** [comment]
This would over complicate the tool. An easier method would be to ignore case while comparing the output with the expected output.

**Comment by cubercsl** [comment]
> This would over complicate the tool. An easier method would be to ignore case while comparing the output with the expected output.

Yes, may be. But I think we should add an option whether to compare in case sencitive mode and float torrance.

**Comment by infalmo** [comment]
Is this good (for floating comparisions)
![image](https://user-images.githubusercontent.com/45997896/78661425-64fae280-78ec-11ea-8b05-9155318799e8.png)

And case insensitive mode
![image](https://user-images.githubusercontent.com/45997896/78661808-071aca80-78ed-11ea-825c-b8eeef84df11.png)

**Comment by cubercsl** [comment]
> Is this good (for floating comparisions)
> ![image](https://user-images.githubusercontent.com/45997896/78661425-64fae280-78ec-11ea-8b05-9155318799e8.png)
> 
> And case insensitive mode
> ![image](https://user-images.githubusercontent.com/45997896/78661808-071aca80-78ed-11ea-825c-b8eeef84df11.png)

It looks very good~
